src/server/pfs/server/driver.go

```diff
@@ -1033,6 +1032,9 @@ func (d *driver) inspectCommit(ctx context.Context, commit *pfs.Commit, block bo
 						return fmt.Errorf("Unmarshal: %v", err)
 					}
 				case watch.EventDelete:
+					// Note: this error is detected explicity in
+					// src/server/worker/master.go. If this error text is changed, that
+					// file should be changed too
```

Comment:
I think it make sense to have this error in `src/server/pfs/pfs.go` where we keep other errors that the outside world might depend on.

Reply:
Done